clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1379,8 +1382,9 @@ public void seekToEnd(Collection<TopicPartition> partitions) {
      *             function is called
      * @throws org.apache.kafka.common.errors.InterruptException if the calling thread is interrupted before or while
      *             this function is called
+     * @throws org.apache.kafka.common.errors.AuthenticationException if authentication fails. {@link AuthenticationException}
      * @throws org.apache.kafka.common.errors.AuthorizationException if not authorized to the topic or to the
-     *             configured groupId
+     *             configured groupId. {@link AuthorizationException}
```

Comment:
Are all of these links actually necessary? From what I can tell, the link is provided automatically by the `@throws`. See here: https://kafka.apache.org/0110/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html.

Reply:
Hmm, I see similar `@link` tags in the javadoc. For example, `Deserializer` and `ConsumerConfig` are hyperlinked for the constructor method [here](https://kafka.apache.org/0110/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#KafkaConsumer(java.util.Map,%20org.apache.kafka.common.serialization.Deserializer,%20org.apache.kafka.common.serialization.Deserializer)), and there are corresponding `@link` tags in the source code [here](https://github.com/apache/kafka/blob/bd0146d984dd5df82fb19aa936e8f4ff9ca40030/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L590).

Reply:
Sorry, you're right. It seems the `@throws` tag automatically creates a link. I'll remove them. @ijuma @rajinisivaram please advise if you see an issue with removing them.

Reply:
Sorry, I didn't it like this. I simply meant that we should emphasize that details of when AuthenticationException are described there. Something as simple as "See the exception for more details" is enough. This should be implicit, but lack of docs is so common that people often don't think to look.

Reply:
Thinking about this a bit more, wouldn't we need to make a similar statement for (almost) all `@throws` tags? Wouldn't the existence of the hyperlink implicitly imply that statement? I see very few other places in the javadocs with a similar statement.

Reply:
The reason why it's worth it in this case is that this is a complicated area where we get a lot of questions. The consumer documents in detail many of the potential exceptions that are specific to the consumer, but AuthenticationException applies equally to all clients so it makes sense to centralise the docs in the exception and nudge users in that direction.

Reply:
OK, that's fair. I'll add it for `AuthenticationException` and `AuthorizationException`. Thanks.